### ISSUE TITLE
why set dealwith_lossnan = False  by default

### ISSUE THREAD

**Issue by ghost** [Holmeyoung/crnn-pytorch#47]
@Holmeyoung 
dealwith_lossnan = False # whether to replace all nan/inf in gradients to zero
In the params.py why you set dealwith_lossnan = False?
to handle the problem "Just don't know why, but when i train the net, the loss always become nan after several epoch." should dealwith_lossnan be set as True?

**Comment by ghost** [comment]
@Holmeyoung thanks you so much

**Comment by Holmeyoung** [comment]
Before pytorch v1.2.0, there is something wrong with ctcloss, the gradients will become nan after several epoch. So, to fix this, i have to "replace all nan/inf in gradients to zero". But they fixed it after v1.2.0, and i just keeped this setting.

![image](https://user-images.githubusercontent.com/18281399/79727593-2f5bed80-831f-11ea-9f27-7f7f99677e5b.png)

**Comment by ghost** [comment]
got it, thanks.
